core/src/main/java/com/novoda/noplayer/drm/DrmSessionCreatorFactory.java

```diff
@@ -18,7 +18,7 @@ public DrmSessionCreator createFor(DrmType drmType, DrmHandler drmHandler) {
                 );
                 return StreamingDrmSessionCreator.newInstance(mediaDrmCallback, new FrameworkMediaDrmCreator());
             case WIDEVINE_MODULAR_DOWNLOAD:
-                return new NoDrmSessionCreator();
+                return new DownloadDrmSessionCreator((DownloadedModularDrm) drmHandler, new FrameworkMediaDrmCreator());
```

Comment:
I think the names aren't particular helpful at the moment, the plan is to try and flatten a bunch of the creators/factories, hopefully that'll help a bunch